handlers/high-charts/src/utils.ts

```diff
@@ -88,3 +88,35 @@ export async function sendOSC(jsonFile: string, outFile: string, server: string,
         })
     ]);
 }
+
+/**
+ * The function will return the string representing the graph title and axes information
+ * @param highChartsData 
+ * @returns graphInfo string
+ */
+export function getGraphInfo(highChartsData: any): string{
+    const chartsData = JSON.parse(JSON.stringify(highChartsData));
+    const title = chartsData.title || chartsData.series[0].name || 'Untitled Chart';
+    const xAxis = chartsData.axes.find((axes: { axis: string; })=>axes.axis == "xAxis");
```

Comment:
Probably not an issue here, but it's generally better to use `===` and `!==` to avoid the [odd effects of type conversion](https://codeahoy.com/javascript/2019/10/12/==-vs-===-in-javascript/). If you aren't relying on this type conversion, I'd recommend `===` throughout these changes.